tests/pfcwd/test_pfcwd_function.py

```diff
@@ -470,6 +470,8 @@ def __init__(self, ptf, router_mac, pfc_params, is_dualtor):
             self.vlan_mac = "00:aa:bb:cc:dd:ee"
         else:
             self.vlan_mac = router_mac
+        # Verify traffic before pfc storm
+        self.verify_rx_ingress("forward")
```

Comment:
Hi, @zhixzhu , Glad to see you again! 
If you want to determine whether the case failure was caused by the case-self or the environmental issue, it's better to add the check in the fixture, then the case would be set-up ERROR instead of case failure if testbed is not OK. 
BTW, if the failure is caused by the previous case which you mentioned, please let me know the previous case name, thanks a lot.